apps/public-docsite-v9/src/Concepts/BuildTimeStyles.stories.mdx

```diff
@@ -0,0 +1,136 @@
+import { Meta, Source, Link } from '@storybook/addon-docs';
+import imageFile from '../../public/style-resolution.png';
+
+<Meta title="Concepts/Developer/Build time styles" />
+
+## Optimize runtime cost of styles
+
+Fluent UI uses the open source [Griffel.js](https://griffel.js.org/) css-in-js styling engine.
+Make sure to take a look at the following docs to get started with styling in Fluent UI:
+
+- [Quick start](?path=/story/quick-start)
+- [Styling components](?path=/story/concepts-developer-styling-components--page)
+
+While there is nothing wrong with the associated runtime costs of a css-in-js engine, larger and more complex
+applications might want to optimize for performance. In Fluent UI, the expensive runtime work only
+happens on the first render of the component.
```

Comment:
This jumps from "happens only once" to "Webpack" too quickly.
1. add one more sentence describing something like "this one-time runtime work can be avoided by processing the styles during build time using *build time transforms*"
2. add a link to the *What is being optimized* below?
3. add a new heading "Setup build time transforms"

Reply:
Good points